frontend/src/app/testQueue/TestCardForm/TestCardFormReducer.tsx

```diff
@@ -173,9 +172,10 @@ export const testCardFormReducer = (
       ) {
         resultState.testResults = updatedResults;
       }
+
       const aoeAnswers = {
         noSymptoms: payload.noSymptoms,
-        symptoms: JSON.stringify(parseRespiratorySymptoms(payload.symptoms)),
+        symptoms: payload.symptoms,
```

Comment:
making this change and pushing the parsing into the `TestCardForm` itself on line 211